packages/core/admin/server/src/bootstrap.ts

```diff
@@ -69,6 +69,43 @@ const syncAPITokensPermissions = async () => {
   }
 };
 
+/**
+ * Ensures the creation of default API tokens during the app creation.
+ *
+ * Checks the database for existing users and API tokens:
+ * - If there are no users and no API tokens, it creates two default API tokens:
+ *   1. A "Read Only" API token with permissions for accessing resources.
+ *   2. A "Full Access" API token with permissions for accessing and modifying resources.
+ *
+ * @sideEffects Creates new API tokens in the database if conditions are met.
+ */
+
+const createDefaultAPITokensIfNeeded = async () => {
+  const userService = getService('user');
+  const apiTokenService = getService('api-token');
+
+  const usersCount = await userService.count();
+  const apiTokenCount = await apiTokenService.count();
+
+  if (usersCount === 0 && apiTokenCount === 0) {
```

Comment:
There might be a better way to check if this is the first startup without adding two queries on every reboot. Like maybe something in core store?

Reply:
What if you manually wipe out your database to reset the data? What about partial data transfer, should the core store flag be transferred as well?

I tried coming out with a better solution but couldn't, if you have an idea please let me know :confused:

Reply:
I'll add a todo about moving the logic to first run completion status and create a task to implement this (or similar) in another PR. Does that make sense?

Reply:
Sounds good to me!